### PR TITLE
Profile settings changes

### PR DESCRIPTION
**Pull request by zbettenbuk**
This PR contains 2 commits:
1. Semantic changes based on post-merge feedback
2. Fix an issue that the introduction of the getPropertyValue caused (no welcome screen video when startWithVideoMuted is true)

### CODE REVIEW

**Comment by lyubomir** [comment]
@virtuacoplenny, please don't merge this PR!

File: react/features/base/profile/functions.js [hunk 1]
```diff
@@ -5,62 +5,69 @@ import { toState } from '../redux';
...
-        && typeof urlParams[urlParamName] !== 'undefined'
-    ) {
-        return urlParams[urlParamName];
+    if (_sources.urlParams) {
+        const urlParams
+            = parseURLParams(state['features/base/connection'].locationURL);
+        const urlParamName = `config.${propertyName}`;
+
+        if (typeof urlParams[urlParamName] !== 'undefined') {
+            return urlParams[urlParamName];
```
Comment (lyubomir) [inline react/features/base/profile/functions.js]: My example source code explicitly avoided the double `urlParams[urlParamName]`.

Comment (zbettenbuk) [reply react/features/base/profile/functions.js]: I didn't get that, sorry, fixed. I hope it's not too much that I combined the entire statement(s)

File: react/features/base/media/middleware.js [hunk 2]
```diff
@@ -60,12 +60,15 @@ function _setRoom({ dispatch, getState }, next, action) {
     // Read the config.
 
     const state = getState();
+    const isRoomValid = Boolean(room);
```
Comment (lyubomir) [inline react/features/base/media/middleware.js]: We actually have a function called `isRoomValid`! I wanted us to use it here but I guess I was vague in my comment. Anyway, I'll handle that.

Comment (zbettenbuk) [reply react/features/base/media/middleware.js]: Oh, I misunderstood, sorry. Thanks for taking care of it!

**Pull request merged**